package.json

```diff
@@ -13,13 +13,12 @@
   "type": "module",
   "exports": {
     ".": {
+      "types": "./dist/types.d.ts",
       "import": "./dist/module.mjs",
       "require": "./dist/module.cjs"
     },
     "./transformers": "./dist/runtime/transformers/index.js",
-    "./transformers/*": "./dist/runtime/transformers/*.js",
-    "./types": "./dist/runtime/types/index.js",
```

Comment:
> These change does not affect current projects.

@farnabaz I think moving the entry of types is a breaking change. 
```ts
import type { ParsedContent, NavItem, MarkdownNode } from '@nuxt/content/dist/runtime/types'
```
does not work any more. Users now need to import directly from `@nuxt/content`

Projects like `nuxt-ui-pro/docs` import many types from `'@nuxt/content/dist/runtime/types'`